contributing-guides/style-guide.md

```diff
@@ -27,12 +27,12 @@ It is run automatically on every pull request,
 but you may install it to test your contributions locally before submitting them:
 
 ```
-npm install tldr-lint
-tldrl -f {{page.md}}
+npm install --global tldr-lint
+tldr-lint -f {{page.md}}
```

Comment:
Your command outputs the valid content in the standard output.
I think it is more useful to document the command that only displays errors:

```suggestion
tldr-lint {{page.md}}
```

See `tldr-lint --help`:
```
A linting tool to validate tldr pages

Options:
  -V, --version         output the version number
  -f, --format          also attempt formatting (to stdout, or as specified by -o)
  -o, --output <file>   output to formatted file
  -i, --in-place        formats in place
  -t, --tabular         format errors in a tabular format
  -v, --verbose         print verbose output
  -i, --ignore <codes>  ignore comma separated tldr-lint error codes (e.g. "TLDR001,TLDR0014")
  -h, --help            display help for command
```